### ISSUE TITLE
Multiple lines per chart

### ISSUE THREAD

**Issue by ubuntudroid** [diogobernardino/WilliamChart#255]
This has been [asked already](https://github.com/diogobernardino/WilliamChart/issues/206) for an older version of the library, but it doesn't seem to be possible with the new version anymore:

Can I display multiple lines (with different colors) in one chart? I didn't find a way to do it using `show()` or `animate`.

**Comment by diogobernardino** [comment]
Not yet on v3.+.

**Comment by ubuntudroid** [comment]
If it's not asked too much: do you happen to have any rough ETA? Btw, I really appreciate your work - the charts look beautiful! ❤️

**Comment by diogobernardino** [comment]
Not really. I honestly need to think whether I want to bring that complexity into the codebase. So far you've been the only person requesting. I will keep the issue open with the `vote up?` label, if enough people show up I might think of.

**Comment by robinpaulson** [comment]
We used a previous version (2.something) to achieve a striped background to the charts, by layering multiple charts on top of each other. It's surprisingly stable and wasn't hugely complicated. Yes, it's clumsy, I wouldn't recommend it for the faint of heart, but maybe it isn't so wrong as you might first think. [Forecastie](https://github.com/martykan/forecastie/).

We may go further and layer all 5 (weather) forecast charts on one set of axes. Or maybe that's *too* much.

**Comment by kkuez** [comment]
Its been over 2 years now, I voted up and am highly interested in such a behaviour. Please have another thought about it and if you personally dont have the time for this, consider calling for external merges.
Kind regards

**Comment by robinpaulson** [comment]
Yeah, I'd like to see it too
